Pod/Classes/SEGAmplitudeIntegration.m

```diff
@@ -152,6 +152,11 @@ - (void)track:(SEGTrackPayload *)payload
 - (void)screen:(SEGScreenPayload *)payload
 {
     if ([(NSNumber *)self.settings[@"trackAllPages"] boolValue]) {
+        [self realTrack:@"Viewed Screen" properties:payload.properties integrations:payload.integrations];
+    } else if ([(NSNumber *)self.settings[@"trackCategorizedPages"] boolValue]) {
+        NSString *event = [[NSString alloc] initWithFormat:@"Viewed %@ Screen", payload.properties[@"category"]];
```

Comment:
I thought if were not deprecating category, this should use the top level category, i.e. payload.category instead?

Reply:
Thinking through this the scope of adding category back into the spec this would require
1. An update to analytics-ios to allow users to set category as a top level property
2. Remove deprecate flag from analytics-android for category
3. Update Mixpanel iOS/Android to check top level category instead of props
4. Update Amplitude iOS/Android to check top level category
5. Update docs

^ And this is just mobile, we would also need to audit `page` behavior across our integrations and see if we have moved over to checking category as a top level prop or within props, and of course add it back into the `page` spec

Reply:
So are you saying we *should* deprecated category? I'm ok with that, but that also involves a fair bit of work.

Reply:
Confirmed in #spec-committee that we will stand by deprecating `category` and expect it to be sent as a property, not a top level trait. We will plan to deprecate the two settings `trackCategorizedPages` and `trackNamedPages` and only maintain the `trackAllPages` functionality, which for mobile we decided on sending a `track` call to Amplitude as a `Viewed Screen` event (instead of `Loaded a Page`).